samples/nested-overlays/src/main/java/com/squareup/sample/nestedoverlays/NestedOverlaysWorkflow.kt

```diff
@@ -53,11 +55,11 @@ object NestedOverlaysWorkflow : StatefulWorkflow<Unit, State, Nothing, Screen>()
         BackButtonScreen(
           ButtonBar(
             Button(
-              name = R.string.close,
+              name = string.close,
```

Comment:
Android Studio strikes again

Reply:
Looks like it's only this file

Reply:
Thanks for catching that. It's just crazy making.